lib/widgets/support_widgets/elegant_body_widet.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+
+class ElegantBody {
```

Comment:
Class names should be a PascalCase of the file's name "elegant_body_widget" for better code quality and maintainability

Reply:
yes will change